packages/fxa-admin-panel/.storybook/main.js

```diff
@@ -4,9 +4,5 @@
 
 module.exports = {
   stories: ['../src/**/*.stories.tsx'],
```

Comment:
Weird thing about fxa-admin-panel: It has a Storybook config, but apparently no stories?